Techome/Techome/Views/AddCaffeine/SearchCaffeineView.swift

```diff
@@ -48,52 +50,64 @@ struct SearchCaffeineView: View {
 }
 
 struct RecentlyAddedCaffeine: View {
+    
     var body: some View {
         
-        
         VStack {
             Text("최근 추가한 카페인")
                 .font(.body)
-                .foregroundColor(.black)
+                .foregroundColor(.customBlack)
                 .fontWeight(.semibold)
                 .frame(maxWidth:.infinity, alignment: .leading)
                 .padding(.horizontal, SearchCaffeineViewLayoutValue.Padding.cardTitleHorizontalPadding)
             
             VStack(spacing: .zero) {
-                ForEach (0 ..< 7) { _ in
-                    CaffeineRecordRow()
+                if IntakeManager.shared.getRecentRecords(count: 6).count != 0 {
+                    
+                    VStack(spacing: .zero) {
+                        ForEach (IntakeManager.shared.getRecentRecords(count: 6), id: \.self) { record in
+                            CaffeineRecordRow(recentRecord: record)
+                        }
+                    }
+                    .background(RoundedRectangle(cornerRadius: SearchCaffeineViewLayoutValue.Radius.card)
+                        .foregroundColor(.white)
+                        .shadow(color: .secondaryShadowGray, radius: SearchCaffeineViewLayoutValue.Radius.shadow, x: .zero, y: .zero)
+                        )
+                } else {
+                    Text("최근 추가된 카페인이 없습니다.")
+                        .padding(.top, 150)
+                        .foregroundColor(.customBlack)
                 }
```

Comment:
```suggestion
                if IntakeManager.shared.getRecentRecords(count: 6).isEmpty {
                    Text("최근 추가된 카페인이 없습니다.")
                        .padding(.top, 150)
                        .foregroundColor(.customBlack)
                } else {
                    VStack(spacing: .zero) {
                        ForEach (IntakeManager.shared.getRecentRecords(count: 6), id: \.self) { record in
                            CaffeineRecordRow(recentRecord: record)
                        }
                    }
                    .background(RoundedRectangle(cornerRadius: SearchCaffeineViewLayoutValue.Radius.card)
                        .foregroundColor(.white)
                        .shadow(color: .secondaryShadowGray, radius: SearchCaffeineViewLayoutValue.Radius.shadow, x: .zero, y: .zero)
                        )
                }
```

!=대신 그대로를 생각해서 처리해도 될 것 같아요 ~

Reply:
앗 그러네요!! 변경할게요 ㅎㅎ